docs/blog/newsletter-december-2020.rst

```diff
@@ -0,0 +1,112 @@
+
+If your manager is generally happy with the status quo but is open to considering your ideas, start with a small example of the changes you'd like to make. This gives you a chance to demonstrate your improvements without spending too much time and effort on work your manager may or may not support.
+
+Finally, customers and other users who are familiar with the existing docs often handle changes quite well if the changes truly improve findability and readability in the docs. The key here is making sure that the changes are clear improvements that justify any inconvenience users experience as they learn the new docs.
+
+-------------------------------------------
+Cleaning up your docs style - automatically
+-------------------------------------------
+​
+Docs linting has been getting a lot of attention on the WTD Slack lately. Here's the most recent collective wisdom from the hivemind!
+
+- `Vale <https://github.com/errata-ai/vale/>`_ seems to be the linting tool of choice, most often supplemented with custom rules or third-party rulesets.
+- A community member has posted videos to help folks get started. See `Documentation as code (Part 3): A Linting How To - The Vale Linter in Action <https://www.youtube.com/watch?v=p1pfwpMv2wU>`__, from Lynette Miles (@esmerel on Slack).
+- Most workflows seem to rely on local linting only, but a few brave souls integrate into an integration pipeline. This seems to be a safer or easier approach if you're building the docs only - that is, if they aren't part of a larger product/code build.
+- One solution to the problem of breaking the build because of spelling errors is to run linting at the commit level. It's a variant of the local approach, but integrated into the larger doc workflow instead of relying on individual local integrations of Vale.
+- Linting can help maintain consistent enforcement of a style guide - for writers as well as non-writers! 
+- It's also useful for an MVP implementation of readability standards. You can lint for simpler words, or simpler grammar.
+- To help you when tidying up resolving merge conflicts, it's a good idea to lint for conflict markers!
+
+------------------------------
+The best time to use a tooltip
+------------------------------
+
+Tooltips and online help are very different tools for providing information to users. How do you choose between them?
+
+The main response from Slack was that it's not a binary choice, and both are useful when implemented well. In fact, they support each other: the brief details in a tooltip can link to online help for fuller details.
+
+Tooltips are simpler for users to access than online help, but they're limited in the amount they can contain. They're at their best when they enhance the interface - for example, showing that a component has a keyboard shortcut. Just be careful, as it's not always easy for users to tell that a tooltip is available.
```

Comment:
limited in the amount of information they can contain? amount of text?

Reply:
Changed!